data/transition-sites/slc.yml

```diff
@@ -0,0 +1,6 @@
+---
+site: slc
+whitehall_slug: student-loans-company
+homepage: https://www.gov.uk/government/organisations/student-loans-company
+tna_timestamp:  20201010101010 # Stub timestamp - site not in TNA
+host: www.slc.co.uk
```

Comment:
Should we add `slc.co.uk` as an alias?

Reply:
yeah - I wasn't sure - they are keeping control of media.slc.co.uk so my original thought was that they might have other subdomains they want to keep around. 
I've added it in - it won't do anything unless they point the apex at us anyway I guess.